pyproject.toml

```diff
@@ -56,9 +57,6 @@ demo = "http://chemprop.csail.mit.edu/"
 [project.scripts]
 chemprop = "chemprop.v2.cli.main:main"
 
-[tool.setuptools_scm]
-write_to = "chemprop/_version.py"
-
```

Comment:
why is this deleted?

Reply:
I just prefer the version to be in the `toml` and there only but we could add this back if there is a consensus. I like the approach I have since it seems more in the spirit of the `toml` file.

Reply:
This is only for local builds, so if you’re installing from source on a new commit then you can have your version reflect that (I.e., “2.0.1+<git_hash>”

Reply:
Oh cool, adding back